### DISCUSSION TITLE
send data from page.jsx to layout.jsx

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#44416] [Help]
hello. 
I load some data in my Page.jsx file from backend. 

some of this data, should be in the part of page, that is in the layout file. 
how can I have data of page file in the layout file?

**Comment by zackdotcomputer** [comment]
The current solution in the docs is to load the data you need again in `layout.jsx` - Next's deduping of `fetch` calls should make it so that you only make the call and load the data once and then it is provided to your components in both places as if shared by a Context.
